HCCGo/app/js/ConnectionService.js

```diff
@@ -872,14 +872,23 @@ connectionModule.factory('connectionService',['$log', '$q', '$routeParams', '$lo
                 }
        });
 
-   }
+   };
 
+   var quickDownload = function(remotePath, localPath) {
+       const sftp = require('ssh2').SFTPStream;
+       sftp.fastGet(remotePath, localPath, function(err) {
```

Comment:
That's not how you get the sftp object.  Look at https://github.com/unlhcc/HCCGo/blob/master/HCCGo/app/js/ConnectionService.js#L520